### PR TITLE
code for shake function

### PR DESCRIPTION
**Pull request by jazheng1**
The first iteration of shake function, where it chooses a random letter in the alphabet 16 times.

### CODE REVIEW

**Comment by jazheng1** [comment]
@jfarmer 
my bad Jesse, did not pay attention and tried to merge to your master, but here is my code for the first iteration of shake.

File: shuffle.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+function shake(){
+  let abcs = 'abcdefghijklmnopqrstuvwxyz'
+  let letterChoice = '';
+  for(let i = 0; i < 16; i++){
+    let randomLetter = abcs.charAt(Math.floor(Math.random() * abcs.length));
+    console.log(randomLetter);
```
Comment (jfarmer) [inline shuffle.js:6]: Remove the random `console.log` statements.

File: shuffle.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+function shake(){
```
Comment (jfarmer) [inline shuffle.js:1]: `shake` is meant to take the `board` as an argument and insert random letters into the `board` at the appropriate positions.

File: shuffle.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  let abcs = 'abcdefghijklmnopqrstuvwxyz'
```
Comment (jfarmer) [inline shuffle.js:2]: `let alphabet = '...';`

File: shuffle.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+  let letterChoice = '';
+  for(let i = 0; i < 16; i++){
+    let randomLetter = abcs.charAt(Math.floor(Math.random() * abcs.length));
```
Comment (jfarmer) [inline shuffle.js:5]: I'd write a function called, say, `randomElement(alphabet)` that encodes this logic. You can index into strings the same way you can arrays, so, e.g., `alphabet[2]` is `'c'`.

**Pull request closed without merging**